Documentation/installation/README.md

```diff
@@ -1,20 +1,25 @@
 # Installation
 
-Clone the git repository and build:
+## Installing the latest stable version
+
+The latest stable version requires go 1.12 or a later version.
+
+With go version 1.16 or later:
 
 ```
-$ git clone https://github.com/go-delve/delve
-$ cd delve
-$ go install github.com/go-delve/delve/cmd/dlv
+$ go install github.com/go-delve/delve/cmd/dlv@latest
 ```
 
-On Go version 1.16 or later, this command will also work:
+With go version 1.15 or older:
+
+Change to any directory that is both outside of your `GOPATH`
+and outside of a module (e.g. a temp directory), and run:
 
 ```
-$ go install github.com/go-delve/delve/cmd/dlv@latest
+$ GO111MODULE=on go get github.com/go-delve/delve/cmd/dlv@latest
```

Comment:
This works for all go 1.12+ and produces reproducible builds. Go 1.12 was released in early 2019, which was 2+ years ago and the method using `git clone` won't work with earlier versions of go any way.